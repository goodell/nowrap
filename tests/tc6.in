123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas vitae imperdiet tellus. Vivamus sagittis mauris in
magna aliquet dapibus. Donec augue libero, commodo ac eleifend sed, ultricies quis lorem. Duis nisl arcu, facilisis a
cursus ac, adipiscing eu metus. Sed cursus enim et metus sodales ornare. Donec elementum ipsum eu augue luctus laoreet.
Duis nec neque eu metus venenatis vestibulum. Aenean arcu arcu, scelerisque fringilla suscipit vitae, mollis ac quam.
Nam eleifend rutrum tellus ac venenatis.

tabs in columns 1..10:
	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
x	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxxxxxx	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
xxxxxxxxx	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

tabs in columns 1..10 with preceding multibyte characters:
	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Ä	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxő	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőxß	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőxßx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőxßxü	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőxßxüx	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
Äxőxßxüxß	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

tabs in columns 1..10 with preceding double-width characters:
	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
x	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日xx	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x本	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x本x	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x本語	*|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x本語x	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
日x本語xx	*89|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

