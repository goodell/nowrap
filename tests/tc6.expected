123456789|123456789|123456789|123456789|123456789|123456789|123456789|12

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas vitae 
magna aliquet dapibus. Donec augue libero, commodo ac eleifend sed, ultr
cursus ac, adipiscing eu metus. Sed cursus enim et metus sodales ornare.
Duis nec neque eu metus venenatis vestibulum. Aenean arcu arcu, sceleris
Nam eleifend rutrum tellus ac venenatis.

tabs in columns 1..10:
	*|123456789|123456789|123456789|123456789|123456789|123456789|12
x	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxxxxxx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
xxxxxxxx	*89|123456789|123456789|123456789|123456789|123456789|12
xxxxxxxxx	*89|123456789|123456789|123456789|123456789|123456789|12

tabs in columns 1..10 with preceding multibyte characters:
	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Ä	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxő	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxőx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxőxß	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxőxßx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxőxßxü	*|123456789|123456789|123456789|123456789|123456789|123456789|12
Äxőxßxüx	*89|123456789|123456789|123456789|123456789|123456789|12
Äxőxßxüxß	*89|123456789|123456789|123456789|123456789|123456789|12

tabs in columns 1..10 with preceding double-width characters:
	*|123456789|123456789|123456789|123456789|123456789|123456789|12
x	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日x	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日xx	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日x本	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日x本x	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日x本語	*|123456789|123456789|123456789|123456789|123456789|123456789|12
日x本語x	*89|123456789|123456789|123456789|123456789|123456789|12
日x本語xx	*89|123456789|123456789|123456789|123456789|123456789|12

